vkvalidator/vkvalidator.vcxproj.user

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I had intentionally added it for the VK_LAYER_PATH setting.  I could get rid of it as it's no skin off my teeth either way.